src/prefect/blocks/notifications.py

```diff
@@ -139,12 +139,38 @@ class MicrosoftTeamsWebhook(AppriseNotificationBlock):
     url: SecretStr = Field(
         ...,
         title="Webhook URL",
-        description="The Teams incoming webhook URL used to send notifications.",
+        description="The Microsoft Power Automate (Workflows) URL used to send notifications to Teams.",
         examples=[
-            "https://your-org.webhook.office.com/webhookb2/XXX/IncomingWebhook/YYY/ZZZ"
+            "https://prod-NO.LOCATION.logic.azure.com:443/workflows/WFID/triggers/manual/paths/invoke?sp=%2Ftriggers%2Fmanual%2Frun&sv=1.0&sig=SIGNATURE"
         ],
     )
 
+    include_image: bool = Field(
+        default=True,
```

Comment:
should this default to true?

Reply:
in this case, yeah I think so! its whether or not the message will show up with the stock image indicating is a message from a "workflows" 
![image](https://github.com/user-attachments/assets/78d0f0ac-a9e3-4483-b1d8-d29df3cc02e5)

Reply:
gotcha, makes sense 👍